sdk/documenttranslation/Azure.AI.DocumentTranslation/tests/TranslationOperationTests.cs

```diff
@@ -0,0 +1,99 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using System.Threading.Tasks;
+using Azure.Storage.Blobs;
+using Azure.Storage.Blobs.Models;
+using Azure.Storage.Sas;
+using NUnit.Framework;
+
+namespace Azure.AI.DocumentTranslation.Tests
+{
+    public class TranslationOperationTests : DocumentTranslationLiveTestBase
+    {
+        public TranslationOperationTests(bool isAsync) : base(isAsync) { }
+
+        private string StorageConnectionString = Environment.GetEnvironmentVariable("STORAGE_ACCOUNT_CONNECTION_STRING");
+
+        private List<string> Documents = new List<string>
+        {
+            "This is the first english test document",
+            "This is the second english test document"
+        };
+
+        private string GetSourceContainerName() => (TestContext.CurrentContext.Test.Name + IsAsync + "source").ToLower();
```

Comment:
Could the name of the containers be variables in the class instead of methods?

Reply:
@maririos It's a function so that if there are multiple tests in the same class, each will have a unique container name.